src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -409,3 +521,13 @@ export const concatAttachmentsAndResponses = (
   response += attachments.reduce((acc, { content }) => acc + content, '')
   return response
 }
+
+/**
+ * Type guard for whether a populated form is email mode
+ * @param form Form document to check
+ */
+export const isEmailModeForm = (
+  form: IPopulatedForm,
+): form is IPopulatedEmailForm => {
+  return form.responseMode === ResponseMode.Email
+}
```

Comment:
move type guards into a types file? I'm also surprised that we aren't able to rely on mongoose discriminators to narrow the types for us, but that's a separate problem.